### ISSUE TITLE
EM&V Use Case

### ISSUE THREAD

**Issue by GamalielL** [hpxmlwg/hpxml#174]
I've put together a list of elements that could be used to perform EM&V analysis for an efficiency program.  I've done this for internal use, but I thought I would share it here as the starting point for a new use case.  Efficiency programs that want to conduct EM&V using data from their HPXML repository could use this to define what elements are required for participating software.  I'm hoping other members with EM&V experience can review it for completeness.  My intention is to capture everything needed for a UMP compliant analysis.

Basic data for weather normalized savings 

- HPXML/Project#/ProjectID/id*
- HPXML/Building#/Site/Address/ZipCode
- HPXML/Project#/ProjectDetails/CompleteDateActual
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionType/Energy/FuelType
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionType/Energy/UnitofMeasure
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionDetail#/Consumption
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionDetail#/StartDateTime
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionDetail#/EndDateTime
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/ConsumptionDetail#/ReadingType 

Data for cross sectional analysis and cost effectiveness

- HPXML/Building#/BuildingDetails/BuildingSummary/BuildingConstruction/ConditionedFloorArea
- HPXML/Building#/BuildingDetails/Systems/HVAC/HVACPlant/HeatingSystem#/AnnualEnergyUse/ConsumptionInfo/ConsumptionType/Energy/FuelType
- HPXML/Project#/ProjectDetails/StartDate
- HPXML/Project#/ProjectDetails/Incentives/Incentive#/IncentiveType/id*
- HPXML/Project#/ProjectDetails/Incentives/Incentive#/FundingSourceCode
- HPXML/Project#/ProjectDetails/Incentives/Incentive#/FundingSourceName
- HPXML/Project#/ProjectDetails/Incentives/Incentive#/IncentiveAmount 
- HPXML/Project#/ProjectDetails/Measures/Measure#/MeasureSystemIdentifiers/SystemIdentifiersInfo#/id*
- HPXML/Project#/ProjectDetails/Measures/Measure#/MeasureCode
- HPXML/Project#/ProjectDetails/Measures/Measure#/Incentives/Incentive#/FundingSourceCode
- HPXML/Project#/ProjectDetails/Measures/Measure#/Incentives/Incentive#/FundingSourceName
- HPXML/Project#/ProjectDetails/Measures/Measure#/Incentives/Incentive#/IncentiveAmount
- HPXML/Project#/ProjectDetails/Measures/Measure#/Cost
- HPXML/Project#/ProjectDetails/Measures/Measure#/InstallingContractor/id*
- 

Data for realization rate calculation (plus calibration data for evaluation)

- HPXML/Project#/ProjectDetails/EnergySavingsInfo#/FuelSavings#/Fuel
- HPXML/Project#/ProjectDetails/Measures/Measure#/EnergySavingsInfo#/FuelSavings#/Units
- HPXML/Project#/ProjectDetails/Measures/Measure#/EnergySavingsInfo#/FuelSavings#/TotalSavings
- HPXML/Project#/ProjectDetails/Measures/Measure#/EnergySavingsInfo#/FuelSavings#/TotalDollarSavings
- HPXML/Project#/ProjectDetails/Measures/Measure#/EnergySavingsInfo#/FuelSavings#/PctReduction
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/BPI2400Inputs/CalibrationQualification
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/BPI2400Inputs/DetailedModelCalibrationHeatingBiasError
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/BPI2400Inputs/DetailedModelCalibrationCoolingBiasError
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/BPI2400Inputs/DetailedModelCalibrationBaseloadBiasError
- HPXML/Consumption#/ConsumptionDetails/ConsumptionInfo#/BPI2400Inputs/SimplifiedModelCalibrationTotalBiasError

**Comment by Tedkidd** [comment]
How about including modeled values for accuracy rankings?

**Comment by GamalielL** [comment]
@Tedkidd That's what's in the last section "Data for realization rate calculation"

**Comment by nmerket** [comment]
@GamalielL Correct me if I'm wrong, but it looks like we would not need to change the schema to accommodate this?

**Comment by GamalielL** [comment]
@nmerket That's correct, but there are probably changes that could be made to improve it, particularly with regards to financial details.  I'm not as familiar with that aspect though.
